hdl/modules/fofb_sys_id/cheby/wb_fofb_sys_id_regs.cheby

```diff
@@ -0,0 +1,44 @@
+memory-map:
+  bus: wb-32-be
+  name: wb_fofb_sys_id_regs
+  description: Interface to FOFB system identification regs
+  x-hdl:
+    busgroup: True
+  children:
+    - block:
+        name: bpm_pos_flatenizer
+        description: Interface to bpm_pos_flatenizer regs
+        size: 16
+        address: 0x00000000
+        children:
+          - reg:
+              name: max_num_cte
+              width: 32
+              access: ro
+              address: next
+              description: |
+                Maximum number of BPM positions that can be flatenized per axis
+              children:
+                - field:
+                    name: val
+                    range: 8-0
+          - repeat:
+              name: axis
+              description: |
+                Two instances of base_idx register, one for each axis (x and y)
+              count: 2
+              children:
+                - reg:
+                    name: base_idx
+                    width: 32
+                    access: rw
+                    address: next
+                    description: |
+                      Together with max_num_cte, defines the valid range of BPM
+                      position indexes, given by [base_idx, base_idx + max_num_cte).
+                      Note that only the P2P BPM positions are being driven to
+                      bpm_pos_flatenizer cores.
+                    children:
+                      - field:
+                          name: val
+                          range: 8-0
```

Comment:
Please change this to use a single register without sub-fields that uses the BPM id number and automatically drive the bpm_flatenaizer correctly for both axis. This makes the interface simpler and eliminates possible bugs in the software layer.